src/main/java/stream/flarebot/flarebot/commands/general/ReportCommand.java

```diff
@@ -36,8 +39,14 @@ public void onCommand(User sender, GuildWrapper guild, TextChannel channel, Mess
                     .stream()
                     .filter(m -> m.getAuthor().equals(user))
                     .collect(Collectors.toList());
-            if (messages.size() > 0)
-                report.setMessages(messages.subList(0, Math.min(5, messages.size() - 1)));
+            if (messages.size() > 0) {
+                messages = messages.subList(0, Math.min(5, messages.size() - 1));
+                List<ReportMessage> reportMessages = new ArrayList<>();
+                for (Message userMessage: messages) {
+                    reportMessages.add(new ReportMessage(userMessage.getContent(), Timestamp.valueOf(userMessage.getCreationTime().atZoneSameInstant(ZoneOffset.UTC).toLocalDateTime())));
```

Comment:
We have a helper method, also very long, split it.

Reply:
What is the helper method?

Reply:
@WalshyDev If your comment is about line 43 then I disagree, it is a nice concise line of code.

Reply:
no i think his comment is about line 46.